base/inc/widgets/base-slider.class.php

```diff
@@ -49,18 +49,39 @@ function initialize() {
 	 */
 	function control_form_fields(){
 		return array(
+			'autoplay' => array(
+				'type' => 'checkbox',
+				'label' => __( 'Autoplay', 'so-widgets-bundle' ),
+				'description' => __( 'Change slides automatically without user interaction.', 'so-widgets-bundle' ),
+				'default' => true,
```

Comment:
@AlexGStapleton Was this perhaps a mistake? Autoplay should most likely be defaulted to false.

Reply:
`Autoplay` set to `true` is how the slider currently works. `false` is new.

 Setting it to false will prevent the slide from automatically changing and will result in a lot of users reporting the slider broke because slides no longer transition automatically anymore.

Reply:
Perfect.